<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupMorph">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>261</y>
      <width>227</width>
      <height>409</height>
     </rect>
    </property>
    <property name="title">
     <string>Basic/Extended</string>
    </property>
    <widget class="QGroupBox" name="groupBoxBasic">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>93</y>
       <width>201</width>
       <height>119</height>
      </rect>
     </property>
     <property name="title">
      <string>Basic</string>
     </property>
     <widget class="QRadioButton" name="radioErode">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>25</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Erode</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioDilate">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>53</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Dilate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBasic">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>81</y>
        <width>175</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxExtended">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>218</y>
       <width>201</width>
       <height>180</height>
      </rect>
     </property>
     <property name="title">
      <string>Extended</string>
     </property>
     <widget class="QRadioButton" name="radioMopTophat">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>100</y>
        <width>175</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>CV_MOP_TOPHAT</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonMopClose">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>25</y>
        <width>175</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>CV_MOP_CLOSE</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonMopGradient">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>75</y>
        <width>175</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>CV_MOP_GRADIENT</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioMopBlackhat">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>125</y>
        <width>175</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>CV_MOP_BLACKHAT</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonMopOpen">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>50</y>
        <width>175</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>CV_MOP_OPEN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonExtended">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>150</y>
        <width>175</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
    <widget class="QSlider" name="horizontalSliderIter">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>48</y>
       <width>191</width>
       <height>39</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelIter">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>25</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Iterations</string>
     </property>
    </widget>
    <zorder>groupBoxExtended</zorder>
    <zorder>groupBoxBasic</zorder>
    <zorder>horizontalSliderIter</zorder>
    <zorder>labelIter</zorder>
   </widget>
   <widget class="QGroupBox" name="groupKern">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>191</width>
      <height>246</height>
     </rect>
    </property>
    <property name="title">
     <string>Kernel</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCols">
       <property name="text">
        <string>Cols:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxCols">
       <property name="minimum">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRows">
       <property name="text">
        <string>Rows:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxRows">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAx">
       <property name="text">
        <string>Anchor x:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxAx">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAy">
       <property name="text">
        <string>Anchor y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxAy">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QRadioButton" name="radioShapeRect">
       <property name="text">
        <string>CV_SHAPE_RECT</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QRadioButton" name="radioShapeCross">
       <property name="text">
        <string>CV_SHAPE_CROSS</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QRadioButton" name="radioShapeEllipse">
       <property name="text">
        <string>CV_SHAPE_ELLIPSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>641</width>
      <height>621</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>639</width>
       <height>619</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLabel" name="labelImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>groupKern</zorder>
   <zorder>groupMorph</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMorph">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuMorph"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBoxCols</tabstop>
  <tabstop>spinBoxRows</tabstop>
  <tabstop>spinBoxAx</tabstop>
  <tabstop>spinBoxAy</tabstop>
  <tabstop>radioShapeRect</tabstop>
  <tabstop>radioShapeCross</tabstop>
  <tabstop>radioShapeEllipse</tabstop>
  <tabstop>horizontalSliderIter</tabstop>
  <tabstop>radioErode</tabstop>
  <tabstop>radioDilate</tabstop>
  <tabstop>pushButtonBasic</tabstop>
  <tabstop>radioButtonMopClose</tabstop>
  <tabstop>radioButtonMopOpen</tabstop>
  <tabstop>radioButtonMopGradient</tabstop>
  <tabstop>radioMopTophat</tabstop>
  <tabstop>radioMopBlackhat</tabstop>
  <tabstop>pushButtonExtended</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
